### PR TITLE
[shopsys] updated acceptance tests because they were no longer working locally

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Acceptance tests were not working on local machines.
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/framework/tests/Test/Codeception/ActorInterface.php [hunk 1]
```diff
@@ -631,13 +631,6 @@ public function moveForward();
      */
     public function moveMouseOver($cssOrXPath = null, $offsetX = null, $offsetY = null);
 
-    /**
-     * @param string $css
-     * @param int|null $offsetX
-     * @param int|null $offsetY
-     */
-    public function moveMouseOverByCss(string $css, ?int $offsetX = null, ?int $offsetY = null): void;
```
Comment (grossmannmartin) [inline packages/framework/tests/Test/Codeception/ActorInterface.php:639]: This is a BC-break

**Review by grossmannmartin — CHANGES_REQUESTED** [review]
can you please also check the failing acceptance tests?

File: open-source-license-acknowledgements-and-third-party-copyrights.md [hunk 2]
```diff
@@ -92,9 +92,9 @@ License: Creative Commons Attribution 3.0 BY
 https://github.com/erikdubbelboer/phpRedisAdmin/blob/master/README.markdown
 
 ### Selenium Docker
-Image: `Selenium/standalone-chrome:3.11`  
-License: Apache License 2.0  
-https://github.com/SeleniumHQ/docker-selenium/blob/master/LICENSE.md
```
Comment (grossmannmartin) [inline open-source-license-acknowledgements-and-third-party-copyrights.md:97]: two spaces are mandatory to preserve formatting 

now it looks like this
Snímek obrazovky 2023-05-05 v 19 15 43

**Review by grossmannmartin — CHANGES_REQUESTED** [review]

File: packages/framework/tests/Test/Codeception/ActorInterface.php (hunk 1 above)
Comment (TomasLudvik) [reply packages/framework/tests/Test/Codeception/ActorInterface.php:639]: I have added upgrade notes for it

File: open-source-license-acknowledgements-and-third-party-copyrights.md (hunk 2 above)
Comment (TomasLudvik) [reply open-source-license-acknowledgements-and-third-party-copyrights.md:97]: fixed

**Comment by TomasLudvik** [comment]
> can you please also check the failing acceptance tests?

That was some weird error and re-running the job fixed it

**Review by grossmannmartin — APPROVED** [review]

**Pull request merged**